packages/components/descriptions/src/description.vue

```diff
@@ -15,7 +15,7 @@
     <div :class="ns.e('body')">
       <table :class="[ns.e('table'), ns.is('bordered', border)]">
         <tbody>
-          <template v-for="(row, index) in getRows()" :key="index">
+          <template v-for="(row, _index) in getRows()" :key="_index">
```

Comment:
Why add `_` prefix?

Reply:
`vue-tsc` reported an unused variable error, and I add the prefix to ignore it.
It could be a issue of `Vue`. You can take a look at this [issue](https://github.com/vuejs/language-tools/issues/329).